etc/env/base/eth_sender.toml

```diff
@@ -19,7 +19,7 @@ max_txs_in_flight=30 # Safe in the local environment, do not repeat on prod (rig
 proof_sending_mode="SkipEveryProof"
 
 # Max L2 blocks to commit in one L1 transaction
-max_aggregated_blocks_to_commit=10
+max_aggregated_blocks_to_commit=1
```

Comment:
revert ?

Reply:
never mind - I misunderstood what this config does.

Reply:
L1 contracts allows max 1 L1 batch committed in a single tx